core-primitives/enclave-api/ffi/src/lib.rs

```diff
@@ -49,6 +49,13 @@ extern "C" {
 		latest_header_size: usize,
 	) -> sgx_status_t;
 
+	pub fn init_shard(
+		eid: sgx_enclave_id_t,
+		retval: *mut sgx_status_t,
+		shard: *const u8,
+		shard_size: u32,
+	) -> sgx_status_t;
```

Comment:
New e-call to explicitly initialize a shard. Before it was a function on the untrusted side, which was a duplicate of the one in the state handler.